pycon/sponsorship/admin.py

```diff
@@ -58,7 +60,9 @@ class SponsorAdmin(admin.ModelAdmin):
     readonly_fields = ["approval_time"]
 
     def contact(self, sponsor):
-        return mark_safe('<a href="mailto:%s">%s</a>' % (escape(sponsor.contact_email), escape(sponsor.contact_name)))
+        # comma-separated emails in mailto: should work: https://www.ietf.org/rfc/rfc2368.txt
+        # but the commas need to be URL-quoted
+        return mark_safe('<a href="mailto:%s">%s</a>' % (escape(quote(','.join(sponsor.contact_emails))), escape(sponsor.contact_name)))
```

Comment:
You can use `format_html`: https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.html.format_html